### ISSUE TITLE
Keep specific hostname when adding domain-based network alert

### ISSUE THREAD

**Issue by achillean** [achillean/shodan-python#168]
If a user runs a command such as:

```shell
shodan alert domain www.shodan.io
```

Then the CLI will create a domain-based network alert for ``shodan.io``. However, it should create a network alert for just ``www.shodan.io`` which is how the [Shodan Monitor](https://monitor.shodan.io) website operates.

**Comment by rmhowe425** [comment]
@achillean Is this issue still something that you would like to be added to the Shodan CLI?

**Comment by achillean** [comment]
Yes, but we probably need to add something like `tldextract` as a dependency to identify whether the user is setting up a specific hostname. The workflow would look something like:

1. Parse the domain with `tldextract`
2. Check if the provided domain has a subdomain
3. If it has a subdomain then lookup the IPv4 and IPv6 addresses for the hostname and use that for the list of IPs. If it doesn't have a subdomain then grab the list of IPs from DNSDB; i.e. what we're currently doing.
4. Create the new asset group with the list of IPs

**Comment by rmhowe425** [comment]
Understood! I'll work on this!
